project/VersionGenerator.scala

```diff
@@ -8,15 +8,15 @@ import sbt._
 import sbt.Keys._
 
 /**
- * Generate version.conf and akka/Version.scala files based on the version setting.
+ * Generate version.conf and Version.scala files based on the version setting.
```

Comment:
should this file be generated as org/apache/pekko/Version.scala?

Reply:
I think only the package name really matters in this case, right? This is just generated code.

Reply:
other unrelated jars on a project's classpath could have Version classes as well - I think it is better to put this class in a  pekko specific package - the akka Version class is in the akka package

Reply:
I'll just change it